layouts/index.html

```diff
@@ -34,7 +34,7 @@
             {{ if gt $nbPosts 0 }}
                 <div class="page-heading">Best posts</div>
                 <ul>
-                {{ range .Data.Pages.ByPublishDate }}
+                {{ range sort .Data.Pages.ByPublishDate "Date" "desc" }}
```

Comment:
Correct me if I'm wrong, but doesn't this produce the same result as `{{ range sort .Data.Pages.ByPublishDate "Date" "desc" }}`, which probably doesn't have two distinct sorting steps.

Reply:
@mtn That is correct, and thanks for pointing this out.  I have pushed a change to fix this. Thanks!